src/main/java/com/gradle/enterprise/bamboo/BuildScanLogScanner.java

```diff
@@ -11,7 +11,7 @@
 final class BuildScanLogScanner extends LogInterceptorAdapter {
 
     private static final Pattern BUILD_SCAN_PATTERN = Pattern.compile("Publishing (build scan|build information)\\.\\.\\.");
-    private static final Pattern URL_PATTERN = Pattern.compile("https?://\\S*");
+    private static final Pattern URL_PATTERN = Pattern.compile("https?://\\S*/s/\\S*");
```

Comment:
for my understanding: I guess we can't restrict more the pattern with the configured GE server URL because it's a generic parsing of any build scan URL (even if not using the injection), correct ?

Reply:
Yes:)